New dbt cloud IA (#2932)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/set-up-sso-google-workspace.md

6. Save the **Consent screen** settings to navigate back to the **Create OAuth client
   id** page.
7. Use the following configuration values when creating your Credentials, replacing `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan.

| Config | Value |
| ------ | ----- |
| **Application type** | Web application |
| **Name** | dbt Cloud |

> | **Authorized Javascript origins** | `https://cloud.getdbt.com` |

Author: ```suggestion
| **Authorized Javascript origins** | `https://YOUR_ACCESS_URL` |
```

> | **Authorized Redirect URIs** | `https://cloud.getdbt.com/complete/gsuite` |

Author: ```suggestion
| **Authorized Redirect URIs** | `https://YOUR_ACCESS_URL/complete/gsuite` |
```

      account using GSuite auth. Optionally, you may specify a CSV of domains
      which are _all_ authorized to access your dbt Cloud account (eg. `dbtlabs.com, fishtowndata.com`)
    - **Slug**: Enter your desired login slug. Users will be able to log into dbt

>       Cloud by navigating to `https://cloud.getdbt.com/enterprise-login/<login-slug>`. Login slugs must

Author: ```suggestion
      Cloud by navigating to `https://YOUR_ACCESS_URL/enterprise-login/LOGIN_SLUG`, replacing `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/deploy/regions-ip-addresses) for your region and plan. The `LOGIN_SLUG` must
```

    <Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/gsuite/gsuite-sso-cloud-verify.png" title="GSuite verify groups"/>

If the verification information looks appropriate, then you have completed
- the configuration of GSuite SSO. 
-

> 

Author: ```suggestion
- <Snippet src="login_url_note" />
```

    <Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/gsuite/gsuite-sso-cloud-verify.png" title="GSuite verify groups"/>

If the verification information looks appropriate, then you have completed

> the configuration of GSuite SSO. 

Author: ```suggestion
- the configuration of GSuite SSO. 
```

> the configuration of GSuite SSO. Members of your team should now be able to log

Author: ```suggestion
- the configuration of GSuite SSO. 
```

-

> **Note:** Changes typically take effect in minutes, but can take up to 24 hours.

Author: ```suggestion
```

> into the dbt Cloud application at `https://cloud.getdbt.com/enterprise-login/<login-slug>`.

Author: ```suggestion
-
```
